apps/src/scratch/scratch.js

```diff
@@ -14,7 +14,7 @@ import { singleton as studioApp } from '@cdo/apps/StudioApp';
 import ScratchView from './ScratchView';
 import { scratchDefaultProject } from './scratchDefaultProject';
 
-const Scratch = window.Scratch = window.Scratch || {};
+const Scratch = window.__TestInterface__ = {};
```

Comment:
__TestInterface__ is something we use for a variety of things. I think what you actually want is
```
const Scratch = {};
window.__TestInterface__ = window.__TestInterface__ || {};
window.__TestInterface__.Scratch = Scratch;
```
or if you want to be ES6y
```
window.__TestInterface__ = {
  ...window.__TestInterface__,
  Scratch
};